gcp_variant_transforms/options/variant_transform_options.py

```diff
@@ -89,10 +91,25 @@ def add_arguments(self, parser):
               'variants. This is useful when there are header fields in '
               'variants not defined in the header sections.'))
 
+  def validate(self, parsed_args):
+    try:
+      # Gets at most one pattern match result of type `filesystems.MatchResult`.
+      first_match = FileSystems.match([parsed_args.input_pattern], [1])[0]
+      if not first_match.metadata_list:
+        raise ValueError(
+            'File pattern {} returned '
+            'no matched files.'.format(parsed_args.input_pattern))
+    except BeamIOError as e:
+      # Retrieves the HttpError from BeamIOError.
+      match_error = e.exception_details[parsed_args.input_pattern]
```

Comment:
match_error is not used?
In any case, I think it's better to throw a more user-friendly error message here...something like "Invalid or inaccessible input pattern {}" b/c the original exceptions from Beam can be cryptic.